.husky/pre-push

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh
+while read local_ref local_sha remote_ref remote_sha; do
+  # Check if the local branch name starts with "docs-release-pr", in which case exit successfully
+  if [[ "$local_ref" =~ ^refs/heads/docs-release-pr-.* ]]; then
+    exit 0
+  fi
+  
+  # Get the list of changed files in this commit
+  changed_files="$(git diff --name-only "${local_sha}" "${remote_sha}")"
+
+  # Check if any of the changed files are in the angular, react, react-native, ui or vue packages
+  if echo "${changed_files}" | grep -E '^packages/angular/|^packages/react/|^packages/react-core/|^packages/react-native/|^packages/ui/|^packages/vue/'; then
```

Comment:
What do you think about targeting `packages/<name>/src`? That way, it would not fail on changes to things like lint configs, tsconfigs, and jest setup that do not changesets.

This will mean there could be false positives, but this might be better than using `--no-verify` on PRs that do not need changesets.

Reply:
Love it! Addressed!